tests/infer/test_inference.py

```diff
@@ -169,26 +160,29 @@ def setUp(self):
         self.alpha0 = Variable(torch.Tensor([1.0]))
         # gamma prior hyperparameter
         self.beta0 = Variable(torch.Tensor([1.0]))
-        self.data = []
-        self.data.append(Variable(torch.Tensor([1.0])))
-        self.data.append(Variable(torch.Tensor([2.0])))
-        self.data.append(Variable(torch.Tensor([3.0])))
+        self.data = Variable(torch.Tensor([[1.0], [2.0], [3.0]]))
         self.n_data = len(self.data)
-        sum_data = self.data[0] + self.data[1] + self.data[2]
-        self.alpha_n = self.alpha0 + sum_data  # posterior alpha
+        data_sum = self.data.sum(0)
+        self.alpha_n = self.alpha0 + data_sum  # posterior alpha
         self.beta_n = self.beta0 + \
             Variable(torch.Tensor([self.n_data]))  # posterior beta
         self.log_alpha_n = torch.log(self.alpha_n)
         self.log_beta_n = torch.log(self.beta_n)
 
+    @pytest.mark.skipif(not dist.gamma.reparameterized, reason='not implemented')
+    def test_elbo_reparameterized(self):
+        self.do_elbo_test(True, 10000)
```

Comment:
did you change any of these step counts?

Reply:
see below, the only change was a 10001 -> 10000 in another test